int-test/expected/PLANCK-ERR.txt

```diff
@@ -7,3 +7,1289 @@ WARNING: planck.io/spit is deprecated. at line 1
 WARNING: planck.io/spit is deprecated. at line 1 
 WARNING: planck.io/spit is deprecated. at line 1 
 WARNING: Use of undeclared Var test-load-file.error-in-file/invalid at line 3 
+Evaluating (println "hi")
```

Comment:
Perhaps for tests involving the `-v` option, `stderr` could be redirected to `/dev/null`